blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -118,7 +124,13 @@ export class MultiBranch extends Component {
                         })
                         }
                     </Table>
+                    {branches.$pager &&
+                        <button disabled={!branches.$pager.hasMore} className="btn-show-more btn-secondary" onClick={() => branches.$pager.fetchMore()}>
```

Comment:
Just looks a bit weird when some of the properties have a `$` prefix and others don't. Makes me wonder why.